pubsub/cloud-client/src/main/java/com/example/pubsub/SubscriberExample.java

```diff
@@ -57,9 +68,10 @@ public static void main(String... args) throws Exception {
       subscriber.startAsync().awaitRunning();
       // Continue to listen to messages
       while (true) {
-        PubsubMessage message = messages.take();
-        System.out.println("Message Id: " + message.getMessageId());
-        System.out.println("Data: " + message.getData().toStringUtf8());
+        MessageAndConsumer messageAndConsumer = messages.take();
```

Comment:
nit: might be worth renaming `messages` to `queue` or something a little more correct